crates/cli/tests/cli_run.rs

```diff
@@ -900,7 +900,7 @@ mod cli_run {
             &[],
             &[],
             &[],
-            "30256\n",
+            "30461\n",
```

Comment:
I guess we probably should change this test long term to not just be a change detector...